packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxFlyoutMenu/OnyxFlyoutMenu.stories.ts

```diff
@@ -14,7 +15,7 @@ const meta: Meta<typeof OnyxFlyoutMenu> = {
   title: "Navigation/modules/FlyoutMenu",
   ...defineStorybookActionsAndVModels({
     component: OnyxFlyoutMenu,
-    events: ["update:open"],
+    events: [],
```

Comment:
Did you remove this intentionally?

Reply:
Yes, otherwise it behaves incorrectly, because the managed mode is overwritten. Will address this in #1725.